mobile-widgets/qmlmanager.cpp

```diff
@@ -869,13 +869,11 @@ void QMLManager::commitChanges(QString diveId, QString date, QString location, Q
 		d->suit = strdup(qPrintable(suit));
 	}
 	if (myDive->buddy() != buddy) {
-		if (myDive->buddy().contains(",")) {
-			if (!buddy.contains(tr("Multiple Buddies"))) {
-				diveChanged = true;
-				free(d->buddy);
-				d->buddy = strdup(qPrintable(buddy));
-			}
-		} else {
+		if (buddy.contains(",") && !buddy.contains(QRegExp("(,\\s)"))){
+			QStringList tempList = buddy.split(',');
+			buddy = tempList.join(", ");
+		}
```

Comment:
and then it re-assembles them with an additional space after the comma, so my list from before becomes "Linus, Joe,  John" with two spaces before "John". Is that what you wanted? And why?